python/cuvs/cuvs/neighbors/prefilters/prefilters.pxd

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# cython: language_level=3
+
+from libc.stdint cimport uintptr_t
+
+
+cdef extern from "cuvs/neighbors/prefilters.h" nogil:
```

Comment:
I know i Menitoned this above, but i'd prefer for us not to have a separate header file for the fitlers. Please merge the contents of `prefilters.h` with `common.h`.